### PR TITLE
Modified Kitchen Sink Page

### PR DESCRIPTION
**Pull request by danclaytondev**
Added data-closable attribute to closable callout div so it closes properly

That's all!

### CODE REVIEW

**Comment by kball** [comment]
Thanks for the PR! This makes the callout closable, but to get it to close, we need to make the close button also have a `data-close` attribute so it will trigger the appropriate event.

**Comment by danclayton50** [comment]
Thanks, done that now and it works!

**Review by IamManchanda — APPROVED** [review]

File: docs/pages/kitchen-sink.md [hunk 1]
```diff
@@ -302,8 +302,8 @@ description: Everything but.
 </div>
 
 ```html_example
-<div class="callout">
-  <button class="close-button" aria-label="Close alert" type="button">
+<div class="callout" data-closable>
+  <button class="close-button" aria-label="Close alert" type="button" data-close>
     <span aria-hidden="true">&times;</span>
   </button>
   <p>This is a static close button example.</p>
```
Comment (IamManchanda) [inline docs/pages/kitchen-sink.md]: I think you need to remove the word `static` ? No ?

**Review by IamManchanda — CHANGES_REQUESTED** [review]

**Comment by danclayton50** [comment]
Removed the word static.

**Comment by kball** [comment]
Perfect! Merged, thanks @danclayton50!

**Pull request merged**